src/api/public/apidocs-new/components/parameters/user_login.yaml

```diff
@@ -0,0 +1,7 @@
+in: path
+name: user_login
+schema:
+  type: string
+required: true
+description: User login
```

Comment:
User or person? We should make up our mind.

Reply:
I think I'm going to use "person" and "people" for consistency in the descriptions and this parameter as well.
@danidoni  what about renaming this file to simply `login`? The description can be something like `Person login`.

I'll do the same in #10658.